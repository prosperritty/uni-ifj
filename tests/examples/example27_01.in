a
10
64.45
