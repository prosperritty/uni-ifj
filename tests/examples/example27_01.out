a
10
0x1.01ccccccccccdp6
0x1p5
32
str
3
strstr
str
0
116
A
